bin/install-dependencies.sh

```diff
@@ -18,7 +18,7 @@ pycsw_tag='2.4.0'
 pipopt='--exists-action=b'
 
 $pip install $pipopt -U numpy==1.16.4
-$pip install $pipopt -U setuptools_rust
+$pip install $pipopt -U setuptools_rust pyOpenSSL
```

Comment:
And here?

Reply:
decided to remove it as no longer needed after pip upgrade